runtime/runtime-params-estimator/src/vm_estimator.rs

```diff
@@ -48,12 +45,9 @@ fn call(code: &[u8]) -> (Option<VMOutcome>, Option<VMError>) {
 
     let promise_results = vec![];
 
-    let mut hash = DefaultHasher::new();
-    code.hash(&mut hash);
-    let code_hash = hash.finish().to_le_bytes().to_vec();
+    let code = ContractCode::new(code.to_vec(), None);
```

Comment:
Not sure whether this is important or not, but this now use the correct sha256 hash and not SipHash (which is what DefaultHasher currently is).

Reply:
Your change is the correct behavior